Description: Do not require neither wget no curl to build
Author: Reinhard Tartler <siretart@debian.org>
Last-Update: 2014-09-27

--- handbrake.orig/make/configure.py
+++ handbrake/make/configure.py
@@ -977,6 +977,7 @@ class SelectTool( Action ):
         self.name     = name
         self.pool     = pool
         self.kwargs   = kwargs
+        self.selected = None
 
     def _action( self ):
         self.session = []
@@ -1428,7 +1429,7 @@ try:
         xcodebuild = ToolProbe( 'XCODEBUILD.exe', 'xcodebuild', abort=False )
         lipo       = ToolProbe( 'LIPO.exe',       'lipo', abort=False )
 
-        fetch = SelectTool( 'FETCH.select', 'fetch', ['wget',wget], ['curl',curl] )
+        fetch = SelectTool( 'FETCH.select', 'fetch', ['wget',wget], ['curl',curl], abort=False ) 
 
     ## run tool probes
     for tool in ToolProbe.tools:
